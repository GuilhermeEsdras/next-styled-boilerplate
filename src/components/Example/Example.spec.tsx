import { renderWithTheme } from '@utils/tests/helpers';

import Example from './Example';

describe('<Example /> Snapshot Matches', () => {
  it('renders Example unchanged', () => {
    const { container } = renderWithTheme(<Example />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
