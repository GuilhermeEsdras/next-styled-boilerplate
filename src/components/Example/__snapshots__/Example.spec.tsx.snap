// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> Snapshot Matches renders Example unchanged 1`] = `
.c0 {
  color: #fff;
}

<main
  class="c0"
>
  <h1>
    Example
  </h1>
</main>
`;
